editor/layout/index.js

```diff
@@ -48,6 +49,7 @@ function Layout( { mode, isSidebarOpened, notices, ...props } ) {
 				<MetaBoxes />
 			</div>
 			{ isSidebarOpened && <Sidebar /> }
+			<Slot name="Editor.Modal" fillChildProps={ { isSidebarOpened } } />
```

Comment:
You noted a comment here: https://github.com/WordPress/gutenberg/pull/2659#issuecomment-332750653

>I'd be happy to do it for V1. The two approaches are much of a muchness, code wise. Removing the modal would feel kind of nice since it'd mean one less <Slot> in the app.

Should we be removing this?

Reply:
Oops! Fixed in 883c7835